tree/20_cfe_basics/packages.cf

```diff
@@ -24,6 +24,14 @@ body package_module test
       #default_options =>  {};
 }
 
+# Not upstream
+body package_module zypper_pattern
+{
+      query_installed_ifelapsed => "$(ncf_def.package_module_query_installed_ifelapsed)";
+      query_updates_ifelapsed => "$(ncf_def.package_module_query_updates_ifelapsed)";
+      #default_options =>  {};
```

Comment:
why it is prefixed by a dash ?

Reply:
It's a comment but Github doesn't know about CFEngine syntax.

Reply:
(all package modules have this field commented, I wanted to stay consistent)